File: src/main/java/io/beaniejoy/dongnecafe/cafe/repository/CafeMenuRepository.java
package io.beaniejoy.dongnecafe.cafe.repository;

import io.beaniejoy.dongnecafe.cafe.domain.CafeMenu;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface CafeMenuRepository extends JpaRepository<CafeMenu, UUID> {
}

File: db/src/main/kotlin/io/beaniejoy/dongnecafe/cafe/repository/CafeMenuRepository.kt
package io.beaniejoy.dongnecafe.cafe.repository

import io.beaniejoy.dongnecafe.cafe.entity.CafeMenu
import org.springframework.data.jpa.repository.JpaRepository

interface CafeMenuRepository : JpaRepository<CafeMenu, Long> {
    fun findByNameAndCafeMenuCategory_Id(name: String, menuCategoryId: Long) : CafeMenu?
}